ext/pcntl/pcntl.c

```diff
@@ -1197,8 +1201,8 @@ PHP_FUNCTION(pcntl_getpriority)
 				php_error_docref(NULL, E_WARNING, "Error %d: No process was located using the given parameters", errno);
 				break;
 			case EINVAL:
-				php_error_docref(NULL, E_WARNING, "Error %d: Invalid identifier flag", errno);
-				break;
+				zend_argument_value_error(2, "must be a valid identifier flag");
```

Comment:
Could we list the identifiers here? As far as I see they are PRIO_PGRP, PRIO_USER, PRIO_PROCESS. If so, then could we validate the values in advance?

Reply:
Yeah, the PRIO_* constants are the possible values here.

Reply:
I think we should also add a test for this and the unshare error condition.

Reply:
Should be done now!